README.md

```diff
@@ -28,207 +41,439 @@ Beyond benchmarking, this solution also **provides guidance on:**
+        required_java_version: "11"
+        
+        # HammerDB Configuration
+        hammerdb_version: 4.12
+        
+        # TNS Configuration
+        default_tns_entry: "ORALAB_STANDALONE"
+        oracle_tns_entries:
+          - tns_name: "ORALAB"
+            host: oralab-scan.oracle-rac.openinfra.lab  # Your Oracle RAC SCAN
+            port: "1521"
+            sid: "pdb1"                                  # Your PDB name
+          - tns_name: "ORALAB_STANDALONE" 
+            host: oralab-standalone1.oracle-rac.openinfra.lab  # Your standalone host
+            port: "1521"
+            sid: "pdb1"
+        
+        # System Configuration
+        benchmark_base_path: /opt/ocpv-benchmark
+        scripts_base_path: /opt/ocpv-benchmark/scripts
+        tns_admin_path: /opt/ocpv-benchmark/tns
+        system_user: cloud-user
+        system_group: cloud-user
+        
+        # Oracle Client Configuration
+        oracle_major_version: 19.26
+        oracle_minor_version: 0.0.0-1.el8
+        oracle_home_path: /usr/lib/oracle/19.26/client64
 ```
- 
-3. **Oracle Client Installation**
-   Specify Oracle Instant Client RPMs. Default is 19.26, but configurable.
 
-```
-    oracle_major_version=19.26
-    oracle_minor_version=0.0.0-1.el8
-    base_url=https://yum.oracle.com/repo/OracleLinux/OL8/oracle/instantclient/x86_64/getPackage
-    oracle_home_path=/usr/lib/oracle/19.26/client64
-``` 
-   
-4. **HammerDB Setup**
-   Define HammerDB version and installation paths. Current default is 4.12.
-```
-      hammerdb_version=4.12
-      hammerdb_base_path=/opt/HammerDB
-      oracle_client_home=/usr/lib/oracle/19.26/client64
-```
-5. **Update Environment Variables for HamerDB Script**
+#### Benefits of Consolidated Variables:
+✅ **Single Source of Truth**: All configuration in one file  
+✅ **Easier Maintenance**: No separate variable files to manage  
+✅ **Better Organization**: Variables alongside inventory definition  
+✅ **Simplified Structure**: No `vars/` directory needed
+
+### 3. Deploy with Selective Tool Installation
 
-You have .env file to set up all the environment variables required for the benchmark.
-Please update below environemnt variables:
+**NEW**: Use the **conditional playbook** with the `benchmark_tool` variable for selective installation:
 
 ```bash
-#password for oracle system user.
-export ORACLE_SYSTEM_PASSWORD=yourpassword
-#and if required any other variables  
+cd ansible
 
-```
+# Test connectivity first
+ansible -i inventory.yaml -m ping oracle_benchmark_client_vms
 
+# Deploy based on benchmark_tool setting in inventory.yaml
+ansible-playbook -i inventory.yaml main_setup_conditional_benchmark.yml
 
-### 2. Test & Run Ansible Playbooks
+# Override at runtime to install specific tools
+ansible-playbook -i inventory.yaml main_setup_conditional_benchmark.yml -e benchmark_tool=hammerdb
```

Comment:
I think we don't required to mention `hammerdb` also can we mention some explanation here that default installation is hammerdb WDYT ?